scripts/win-installer/setup.iss

```diff
@@ -2,7 +2,7 @@
 ; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
 
 #define MyAppName "Juju"
-#define MyAppVersion "2.9.28"
+#define MyAppVersion "{#GetEnv('JUJU_VERSION')}"

```

Comment:
Can we use a syntax to default to "2.9.28" (or whatever it is set to) if JUJU_VERSION is not set

Maybe

```
#if GetEnv('JUJU_VERSION') != ""
#define MyAppVersion=GetEnv('JUJU_VERSION')
#else
#define MyAppVersion="2.9.28"
#endif
```